test/testdata/packager/package-must-be-strict/__package.rb

```diff
@@ -1,6 +1,6 @@
+  # typed: true
+# ^^^^^^^^^^^^^ error: Package files must be at least `# typed: strict`
```

Comment:
This was previously reported with location `{0, 0}` but I've tweaked it to use the identified location of the `typed:` sigil. That also means we can offer an autocorrect, although I haven't in this PR.